packages/core/app-extensions/src/form/formModifier.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * remove create action group from main action bar
+ */
+export const removeCreate = formDefinition => ({
+  ...formDefinition,
+  children: formDefinition.children.map(rootItem => {
+    if (rootItem.id === 'main-action-bar') {
```

Comment:
I would prefer to have a `constants.js` file which includes all these special, const values. (e.g. `main-action-bar`, `output`). It will be easier to find and change them if needed.